romancal/resample/resample_utils.py

```diff
@@ -102,3 +103,30 @@ def make_output_wcs(
     )
 
     return wcs, pscale * 3600.0, pscale_ratio
+
+
+def compute_var_sky(model) -> None:
+    """
+    Add sky variance array to a datamodel.
+
+    Parameters
+    ----------
+    model : `ImageModel`
+        A datamodel to which the sky variance array will be added.
+
+    Returns
+    -------
+    None
+    """
+
+    dnu = (model["dq"] & pixel.DO_NOT_USE) != 0
+    median_data = np.median(model["data"][~dnu])
+    ok_data = model["data"] != 0
+
+    var_sky = model["var_rnoise"].copy()
+    var_sky[ok_data] = (
+        model["var_rnoise"][ok_data]
+        + model["var_poisson"][ok_data] / model["data"][ok_data] * median_data
+    )
```

Comment:
This doesn't match the docstring:
```
VAR_SKY = (VAR_RNOISE + VAR_POISSON) / (DATA * MEDIAN(DATA))
```
Whereas this code is:
```
VAR_SKY = (VAR_RNOISE + VAR_POISSON / DATA * MEDIAN(DATA))
```
Which one should it be?

Reply:
It should be the latter: ``VAR_SKY = VAR_RNOISE + VAR_POISSON / DATA * MEDIAN(DATA)``